src/dagmc/DagMC.hpp

```diff
@@ -212,44 +212,223 @@ class DagMC {
+   * location.
+   * @param[out] result The distance from the location to the closest point in
+   * the volume.
+   * @param[out] surface Optional. The handle of the surface on which the
+   * closest point lies. Default is 0.
+   *
+   * @return Returns an ErrorCode indicating the success or failure of the
+   * operation.
+   */
   ErrorCode closest_to_location(EntityHandle volume, const double point[3],
                                 double& result, EntityHandle* surface = 0);
 
+  /**
+   * @brief Measures the volume of a specified volume.
+   *
+   * @param volume The volume to be measured.
+   * @param[out] result The measured volume.
+   *
+   * @return Returns an ErrorCode indicating the success or failure of the
+   * operation.
+   */
   ErrorCode measure_volume(EntityHandle volume, double& result);
 
+  /**
+   * @brief Measures the area of a specified surface.
+   *
+   * @param surface The surface to be measured.
+   * @param[out] result The measured area.
+   *
+   * @return Returns an ErrorCode indicating the success or failure of the
+   * operation.
+   */
   ErrorCode measure_area(EntityHandle surface, double& result);
 
+  /**
+   * @brief Determines the sense of one or more surfaces with respect to a
+   * specified volume.
+   *
+   * This method assumes that the surfaces passed in are part of the volume.
+   *
+   * @param volume The volume with respect to which the sense is determined.
+   * @param num_surfaces The number of surfaces for which to determine the
+   * sense.
+   * @param surfaces An array of handles of the surfaces for which to determine
+   * the sense.
+   * @param[out] senses_out An array in which to store the determined senses.
+   *
+   * @return Returns an ErrorCode indicating the success or failure of the
+   * operation.
+   */
   ErrorCode surface_sense(EntityHandle volume, int num_surfaces,
                           const EntityHandle* surfaces, int* senses_out);
 
+  /**
+   * @brief Determines the sense of a surface with respect to a specified
+   * volume.
+   *
+   * This method assumes that the surface passed in is part of the volume.
+   *
+   * @param volume The volume with respect to which the sense is determined.
+   * @param surface The handle of the surface for which to determine the sense.
+   * @param[out] sense_out The determined sense.
+   *
+   * @return Returns an ErrorCode indicating the success or failure of the
+   * operation.
+   */
   ErrorCode surface_sense(EntityHandle volume, EntityHandle surface,
                           int& sense_out);
 
+  /**
+   * @brief Gets the angle between the normal of a specified surface and a ray
+   * from the ray origin in a specified direction.
+   *
+   * @param surf The surface with respect to which the angle is determined.
+   * @param xyz A 3-element array representing the coordinates of the point.
+   * @param angle[out] A 3-element array in which to store the determined angle.
```

Comment:
```suggestion
   * @brief Gets the normal vector of the surface at the specified point of
   * intersection by a ray from the ray origin in a specified direction.
   *
   * @param surf The surface with respect to which the normal vector is determined.
   * @param xyz A 3-element array representing the coordinates of the point of 
   * intersection.
   * @param normal[out] A 3-element array in which to store the determined normal vector.
```

Reply:
I don't expect to change the name of the output parameter in this PR since it's only meant to update comments/documentation.

Reply:
Is this right?